docs/docs/running-node/auth.md

```diff
@@ -0,0 +1,49 @@
+# Authentication and authorization
+
+Bacalhau includes a flexible auth system that supports multiple methods of auth
+that are appropriate for different deployment environments.
+
+## By default
+
+With no specific authentication configuration supplied, Bacalhau runs in
+"anonymous mode" – which allows unidentified users limited control over the
+system. "Anonymous mode" is only appropriate for testing or evaluation setups.
+
+In anonymous mode, Bacalhau will allow:
+
+- Users identified by a self-generated private key to submit any job, cancel
+  their own jobs, read job lists and use `bacalhau describe`.
+- Users not identified by any key to access other read-only endpoints, such as
+  node or agent information.
+
+## Restricting anonymous access
+
+Bacalhau auth is controlled by policies. Configuring the auth system is done by
+just supplying a different policy file.
+
+Restricting API access to only users that have authenticated requires specifying
+a new **authorization policy**. You can download a policy that restricts
+anonymous access and install it by using:
+
+    curl -sL https://raw.githubusercontent.com/bacalhau-project/bacalhau/main/pkg/authz/policies/policy_ns_anon.rego -o ~/.bacalhau/no-anon.rego
```

Comment:
we'll need to be careful to never move or renaming `policy_ns_anon.rego`. Maybe it would be better to reference this via a commit or a version? I don't really have a good suggestion here, just worried that it would be easy for docs to become out of data with this reference.

Reply:
Or perhaps we could embed this rego policy in the binary? :thinking:

Reply:
I guess with a specific commit we'd then need to remember to update this if we make fixes or improvements. Either way it's just a careful review that is needed. Perhaps we can introduce an automated test or something?

Yeah, I don't actually imagine this being the long term user experience for doing this – but I think it's appropriate right now for beta users. I can imagine in the future we'd have explicit commands to set up users and things, or at least a dedicated tool, but we need to know more about what auth types people commonly want to use.

Is there any value in having something like `bacalhau auth install policy_ns_anon` or something? Just seems to be a bit overkill for where we're at?

Reply:
> Is there any value in having something like bacalhau auth install policy_ns_anon or something? Just seems to be a bit overkill for where we're at?

seems like a good thing to keep in our back pocket for later, I think you are right - overkill for now.